packages/core/src/helpers/multipartContent.ts

```diff
@@ -0,0 +1,90 @@
+import { bytes, BytesLike } from '@ckb-lumos/codec';
+import parseMultipartMessage, { encodeMultipartMessage } from '@exact-realty/multipart-parser';
+import { TDecodedMultipartMessage } from '@exact-realty/multipart-parser/dist/encodeMultipartMessage';
+import { TMultipartMessageGenerator } from '@exact-realty/multipart-parser/dist/parseMultipartMessage';
+import { TTypedArray } from '@exact-realty/multipart-parser/dist/types';
+import { bufferToRawString, bytifyRawString } from './buffer';
+
+export type AsyncableIterable<T> = AsyncIterable<T> | Iterable<T>;
+
+export type ResolvedMultipartContent = {
+  headers: Headers;
+  body?: Uint8Array | null;
+  parts?: ResolvedMultipartContent[] | null;
+};
+
+export async function decodeMultipartContent(message: string, boundary: string): Promise<ResolvedMultipartContent[]> {
```

Comment:
If there's no boundary text included, what will happen?

Reply:
> If there's no boundary text included, what will happen?

Do you mean like this? The below example will just throws an "invalid message" error:

```ts
decodeMultipartContent('text without boundary', 'boundary')
```

Reply:
Shall there be a detector func to check whether this is a valid multipart?

Reply:
> Shall there be a detector func to check whether this is a valid multipart content data?

Added simple APIs for testing decodability: `isMultipartContentValid`, `isMultipartContentAsBytesValid`